packages/manager/apps/key-management-service/src/components/credential/create/identities/tile/IdentitiesServiceAccountTile.component.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { Dispatch, SetStateAction, useEffect, useState } from 'react';
+import { OsdsText } from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { IdentityOauthClient } from '@/types/identity.type';
+import IdentitiesBaseTile from './IdentitiesBaseTile.component';
+import IdentitiesTileText from './IdentitiesTileText.component';
+
+type IdentitiesServiceAccountTileProps = {
+  serviceAccount: IdentityOauthClient;
+  selectedServiceAccountList: IdentityOauthClient[];
+  setSelectedServiceAccountList: Dispatch<
+    SetStateAction<IdentityOauthClient[]>
+  >;
+};
+const IdentitiesServiceAccountTile = ({
+  serviceAccount,
+  selectedServiceAccountList,
+  setSelectedServiceAccountList,
+}: IdentitiesServiceAccountTileProps) => {
+  const { t } = useTranslation('key-management-service/credential');
+  const [isChecked, setIsChecked] = useState<boolean>(false);
+
+  useEffect(() => {
+    if (
+      selectedServiceAccountList.find(
+        (serviceAccountInList) =>
+          serviceAccountInList.identity === serviceAccount.identity,
+      )
+    ) {
+      setIsChecked(true);
```

Comment:
I think you may need to update the isChecked even if the condition return false

Reply:
It's used on component init, checking that previous selection.
By default they are not selected, false, so it can only turn On.
It can't be changed somewhere else while this is mounted.